util/logging.c

```diff
@@ -17,14 +17,15 @@
 
 #include "logging.h"
 #include "sighdl.h"
+#include "nvme-print.h"
```

Comment:
I would like to avoid pulling in something from the parent directly into the util folder. Looking at what loggin.c does I suggest just move it to the parent folder.